src/components/app_download/AppDownload.module.scss

```
@@ -78,5 +81,10 @@
 }
 .appDownloadButtonContainer {
   display: flex;
-  justify-content: center;
+  @media (max-width: $medium) {
+    justify-content: flex-start;
+  }
+  @media (min-width: $medium) {
+   justify-content: flex-end;
+  }
```

Just a personal opinion, but what is the value of having two media rules here? Could we not just create a fallback value, eliminating some extra code? 

Ie, would it be less confusing to create a single media rule (for mobile, or screens with a max-width of `$medium`), and let the fallback be for the desktop? 

```suggestion
  justify-content: flex-end;
  @media (max-width: $medium) {
    justify-content: flex-start;
  }
```